### DISCUSSION TITLE
private lists

### DISCUSSION THREAD

**Discussion by michael-azogu** [community/community#31135] [Other Feature Feedback, Questions, & Ideas]
Private Lists are very much needed.
I create lists solely for myself & I don't want others to see most of them.

option to make entire list private.

**Comment by melvincwng** [comment]
Hi @michael-azogu, there is a similar discussion thread on private lists over [here](https://github.com/community/community/discussions/8766).

This private list feature seems to be tracked by GitHub as mentioned by the GitHub staff in that thread.

So, let's hope this new feature/enhancement comes out soon 🙏

**Reply by amerops92** [reply]
0
